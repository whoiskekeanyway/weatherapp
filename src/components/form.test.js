import renderer from 'react-test-renderer';
import Form from '../components/form';

it('renders correctly', () => {
  const tree = renderer
  .create(<Form />).toJSON();
  expect(tree).toMatchSnapshot();
});
