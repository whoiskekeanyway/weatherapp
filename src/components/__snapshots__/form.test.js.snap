// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container h-100"
>
  <form>
    <div>
      
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-3 offset-md-2"
      >
        <input
          autoComplete="off"
          className="form-control"
          name="city"
          placeholder="City"
          type="text"
        />
      </div>
      <div
        className="col-md-3 mt-md-0 mt-2 text-md-left "
      >
        <button
          className="btn btn-warning"
        >
          Get Weather
        </button>
      </div>
    </div>
  </form>
</div>
`;
